demo/src/main/java/org/libtorrent4j/demo/DhtShell.java

```diff
@@ -1,20 +1,39 @@
+/*
+ * Copyright (c) 2020, Alden Torres
+ *
+ * Licensed under the terms of the MIT license.
+ * Copy of the license at https://opensource.org/licenses/MIT
+ */
+
 package org.libtorrent4j.demo;
 
-import org.libtorrent4j.*;
-import org.libtorrent4j.alerts.*;
+import org.libtorrent4j.AlertListener;
+import org.libtorrent4j.Ed25519;
+import org.libtorrent4j.Entry;
+import org.libtorrent4j.Hex;
+import org.libtorrent4j.Pair;
+import org.libtorrent4j.SessionManager;
+import org.libtorrent4j.SessionParams;
+import org.libtorrent4j.Sha1Hash;
+import org.libtorrent4j.TcpEndpoint;
+import org.libtorrent4j.alerts.Alert;
+import org.libtorrent4j.alerts.AlertType;
+import org.libtorrent4j.alerts.DhtPutAlert;
+import org.libtorrent4j.alerts.ListenFailedAlert;
+import org.libtorrent4j.alerts.ListenSucceededAlert;
 
 import java.io.File;
+import java.nio.file.Files;
+import java.nio.file.Paths;
 import java.util.ArrayList;
 import java.util.Scanner;
 
 /**
- * @author gubatron
  * @author aldenml
  */
 public final class DhtShell {
 
-    /*
-    public static void main(String[] args) throws Throwable {
+    public static void main(String[] args) {
```

Comment:
Codacy found an issue: [The method 'main(String)' has an NPath complexity of 208, current threshold is 200](https://app.codacy.com/manual/aldenml/libtorrent4j/pullRequest?prid=5897352)